qiskit/dagcircuit/dagcircuit.py

```diff
@@ -1384,6 +1384,7 @@ def filter_fn(node):
             return node.type == 'op' and len(node.qargs) == 1 \
                 and len(node.cargs) == 0 and node.condition is None \
                 and not node.op.is_parameterized() \
+                and not len(node.op.decompositions) == 0
```

Comment:
```suggestion
                and isinstance(node.op, Gate) \
                and hasattr(node.op, '__array__')
```

From https://github.com/Qiskit/qiskit-terra/issues/5543#issuecomment-755710421 .